scripts/paypal/update-hosts-webhooks.ts

```diff
@@ -26,8 +26,11 @@ const getAllHostsWithPaypalAccounts = () => {
 
 const main = async (): Promise<void> => {
   const allHosts = await getAllHostsWithPaypalAccounts();
-  const ignoredSlugs = process.env.SKIP_SLUGS ? process.env.SKIP_SLUGS.split(',') : [];
-  const filteredHosts = allHosts.filter(host => !ignoredSlugs.includes(host.slug));
+  const ignoredSlugs = process.env.SKIP_SLUGS ? process.env.SKIP_SLUGS.split(',') : null;
+  const onlySlugs = process.env.ONLY_SLUGS ? process.env.ONLY_SLUGS.split(',') : null;
+  const filteredHosts = allHosts.filter(host => {
+    return (!ignoredSlugs || !ignoredSlugs.includes(host.slug)) && (!onlySlugs || onlySlugs.includes(host.slug));
+  });
```

Comment:
I suggest decoupling the filter logic for readability, maybe something like:
```suggestion
  const filteredHosts = onlySlugs
    ? allHosts.filter(host => onlySlugs.includes(host.slug))
    : ignoredSlugs
      ? allHosts.filter(host => !ignoredSlugs.includes(host.slug))
      : allHosts;
```

Reply:
Not much time to go back to it right now, but I agree that this logic would be cleaner